crates/backend/src/codegen.rs

```diff
@@ -173,6 +173,7 @@ impl ToTokens for ast::Struct {
         (quote! {
             #[automatically_derived]
             impl #wasm_bindgen::describe::WasmDescribe for #name {
+                #[cfg_attr(feature = "unstable-coverage", allow_internal_unstable(coverage_attribute))]
```

Comment:
```suggestion
                #[coverage(off)]
```
Is fine here, but you have to insert it conditionally depending on `feature = "unstable-coverage"`, so through a variable, e.g. `#coverage`.

On the proc-macro itself you put `allow_internal_unstable`:
```rust
#[proc_macro_attribute]
#[cfg_attr(feature = "unstable-coverage", allow_internal_unstable(coverage_attribute))]
pub fn wasm_bindgen(attr: TokenStream, input: TokenStream) -> TokenStream {
```
Now any library using `wasm-bindgen` with `feature = "unstable-coverage"` can do that as long as they are on nightly without themselves having to use `#![feature(coverage_attribute)]`.